docs/03-tutorials/00-eventing/evnt-03-type-cleanup.md

```diff
@@ -36,29 +36,40 @@ You learn how Eventing behaves when you create a [Subscription](../../05-technic
    
    ```bash
    cat <<EOF | kubectl apply -f -
-     apiVersion: serverless.kyma-project.io/v1alpha1
-     kind: Function
-     metadata:
-       labels:
-         serverless.kyma-project.io/build-resources-preset: local-dev
-         serverless.kyma-project.io/function-resources-preset: S
-         serverless.kyma-project.io/replicas-preset: S
-       name: lastorder
-       namespace: default
-     spec:
-       deps: '{ "dependencies": {}}'
-       maxReplicas: 1
-       minReplicas: 1
-       source: |
-         module.exports = {
-           main: async function (event, context) {
-             console.log("Received event: ", event.data, ", Event Type: ", event.extensions.request.headers['ce-type']);
-             return; 
-           } 
-         }
+   apiVersion: serverless.kyma-project.io/v1alpha2
+   kind: Function
+   metadata:
+     labels:
+       serverless.kyma-project.io/build-resources-preset: local-dev
```

Comment:
Now we rather prefer these settings inside `resourceConfiguration` except `serverless.kyma-project.io/replicas-preset`